src/main/java/frc/robot/subsystems/leds/LEDSubsystem.java

```diff
@@ -38,20 +43,70 @@ public void periodic() {
         // Toggle the blink boolean every duration to swap the LEDs between the driver piece color
         // and the blink color.
         if (blinkTimer.advanceIfElapsed(BLINK_DURATION_SECONDS)) blinking = !blinking;
-        ledStrip.setSolidColor(blinking ? BLINK_COLOR : color);
+        if(manual && continuous){
+            ledStrip.updateContinuousColor(blinking ? BLINK_COLOR : color);
+            ledStrip.setContinuousColor();
+        } else {
+            Color currentColor;
+            if(!aprilTimer.hasElapsed(APRIL_BLINK_DURATION)){
+                currentColor = aprilColor;
+            } else {
+                currentColor = color;
+            }
+            if(blinking){
+                ledStrip.setSolidColor(BLINK_COLOR);
+                ledStrip.updateContinuousColor(currentColor);
```

Comment:
i think a source of confusoin is what `updateContinuousColor` does, and documenting that better might help. Why is it called after setting a solid color? Why is it called twice if blinking?

Reply:
i added documentation, update increments the colors while set actually sets the color. if you want to update the colors but not set them you only call update, otherwise you would call both.